examples/hackernews/package.json

```diff
@@ -9,17 +9,17 @@
   "main": "./dist/index.js",
   "devDependencies": {
     "@types/babel__core": "^7.20.1",
-    "@types/node": "^18.16.19",
+    "@types/node": "^18.17.5",
     "esbuild": "^0.14.54",
-    "solid-start-node": "^0.3.0",
+    "solid-start-node": "workspace:^",
```

Comment:
Doesn't this lead to create-solid cli initalizing broken projects? As far as I know it pulls from the repo. Worth checking.

Reply:
@edivados good catch! This was a pnpm takeover and my mind totally defaulted to the examples shipping with the CLI. 🙈

Reply:
@edivados ready! Thanks for the patience and catch!

Reply:
Cool. Looks like two spots are left.
- node adapter on this file
- solid-start and solid-start-node on with-vitest example

Reply:
@edivados donezies.